mangaki/mangaki/templates/fix/fix_index.html

```diff
@@ -0,0 +1,69 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+
+{% block title %}Liste des suggestions {% if suggestions.paginator.page_range|length > 1 %}- Page {{ suggestions.number }} {% endif %}- Mangaki{% endblock %}
+
+{% block body %}
+
+<h1>Liste des suggestions</h1><br/>
+
+{% for suggestion in suggestions %}
+<div class="row">
+<div class="panel panel-default">
+    <h3 class="panel-heading panel-title">
+        <a href="{% url "fix-suggestion" suggestion.id %}" class="text-primary">Suggestion #{{ suggestion.id }}</a>
+        <span class="small font-weight-bold">
+            &nbsp;&nbsp;<span class="glyphicon glyphicon-{% if suggestion.is_checked %}lock{% else %}flag{% endif %}" aria-hidden="true"></span>
+            Ce problème est {% if suggestion.is_checked %}résolu{% else %}en cours{% endif %}
```

Comment:
It can be rewritten as: `{{ suggestion.is_checked |yesno:'résolu,en cours' }}` (according to [this](https://docs.djangoproject.com/fr/1.11/ref/templates/builtins/#yesno)).